dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/FATSuite.java

```diff
@@ -51,8 +51,8 @@ public class FATSuite {
     public static final String BETA_ID = MicroProfileActions.MP61_ID + "_BETA";
 
     public static RepeatTests aboveMP50Repeats(String serverName) {
-        return TelemetryActions
-                        .repeat(serverName, TelemetryActions.MP50_MPTEL11, MicroProfileActions.MP60, MicroProfileActions.MP61)
+        return MicroProfileActions
+                        .repeat(serverName, TelemetryActions.MP14_MPTEL11,  TelemetryActions.MP41_MPTEL11, TelemetryActions.MP50_MPTEL11, MicroProfileActions.MP60)
                         .andWith(FeatureReplacementAction.EE10_FEATURES().withBeta().fullFATOnly().withID(BETA_ID));
```

Comment:
`MP61` has been removed from this list.

`MP14_MPTEL11` is listed first which means it will be the repeat that runs in lite mode. This probably isn't what we want? I would expect either `MP61` (since we expect that to be most common for telemetry-1.1) or `MP60` (since that's the most recent released version) to be listed first and to run in lite mode.

You're also running `MP60` in beta mode whereas before you were running `MP61`. We may not need the beta repeat any more, I thought that was mostly for testing servlet tracing since that was beta guarded.

Lastly, the name of this method is still `aboveMP50Repeats` - you don't have to change that now, but we should change it at some point.